network-api/networkapi/buyersguide/templates/fragments/publication_table_of_contents.html

```diff
@@ -14,47 +14,47 @@
           <h2>{{ page.contents_title }}</h2>
         </div>
         <div class="col-12">
-          {% for child in page.get_children.live %}
+          {% for child in child_pages %}
             <div class="row publication-row pt-3 pb-sm-4 pb-0">
               <div class="col-auto">
                 <div class="publication-chapter-number">
                   {{ forloop.counter }}
                 </div>
               </div>
-              <div class="col-8 ml-sm-3{% if not child.get_children_count %} d-flex align-items-center{% endif %}">
-                <h3 class="{% if child.get_children_count %}publication-chapter-title{% else %}m-0 {% endif %}">
-                  <a href="{{ child.url }}" class="d-block {% if child.get_children_count %}mb-4{% endif %}">
-                    {{ child.title }}
+              <div class="col-8 ml-sm-3{% if not child.child.get_children_count %} d-flex align-items-center{% endif %}">
+                <h3 class="{% if child.child.get_children_count %}publication-chapter-title{% else %}m-0 {% endif %}">
+                  <a href="{{ child.child.url }}" class="d-block {% if child.child.get_children_count %}mb-4{% endif %}">
+                    {{ child.child.title }} {% if child.child.has_unpublished_changes %}🐣{% endif %}
                   </a>
                 </h3>
-                {% for grandchild in child.get_children.live %}
+                {% for grandchild in child.grandchildren %}
```

Comment:
single child would become `entry`:
```suggestion
                {% for grandchild in entry.grandchildren %}
```